ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -66,19 +66,16 @@ Loader {
 
     // These 2 properties can be dropped when the new unfurling flow supports GIFs
     property var links
-    readonly property var localUnfurlLinks: {
+    readonly property var gifLinks: {
         if (!links)
             return []
         const separator = " "
         const arr = links.split(separator)
         const filtered = arr.filter(value => {
-                                        const v = value.toLowerCase()
-                                        return localAccountSensitiveSettings.gifUnfurlingEnabled && value.endsWith('.gif')
+                                        return value.toLowerCase().endsWith('.gif')
                                     })
-
-
-        const out = filtered.join(separator)
-        return out
+        console.log("<<< calculating gifLinks:", filtered)
```

Comment:
Leftovers?

Reply:
removed in 531a8cb1215bcc377bf37966ebdb26bc20880589 🙂